util/CMakeLists.txt

```diff
@@ -9,4 +9,8 @@
 # See the COPYING file for more information.
 ################################################################################
 
-add_subdirectory(geosop)
+option(BUILD_GEOSOP "Build geosop command-line interface" ON)
```

Comment:
How about calling this `GEOS_BUILD_GEOSOP`, so that GEOS options are grouped together?

(And we should really change a couple of the others, like `DISABLE_OVERLAYNG` ==> `GEOS_DISABLE_OVERLAYNG`)

Reply:
I guess there is also an existing pattern for `BUILD_...`.  So could continue with that.

Reply:
yeah, I know what you mean and did briefly consider the two conventions.

Reply:
If it helps, [PROJ's CMake options](https://proj.org/install.html#cmake-configure-options) use a <tt>BUILD_*NAME*</tt> convention for each tool, which is what swayed my decision here. The main reason why `GEOS_`  is prefixed to many CMake variables is to avoid clashes with other variables when creating large CMake projects. No other tool is named geosop, so no clash.